test/test_suite.py

```diff
@@ -682,3 +685,17 @@ def test_unique_constraint_raises(self):
 
         with pytest.raises(spanner_dbapi.exceptions.ProgrammingError):
             self.metadata.create_all()
+
+    @pytest.mark.skip("Spanner doesn't support temporary tables")
+    def test_get_temp_table_indexes(self):
+        pass
+
+    @pytest.mark.skip("Spanner doesn't support temporary tables")
+    def test_get_temp_table_unique_constraints(self):
+        pass
+
+
+class InsertBehaviorTest(_InsertBehaviorTest):
+    @pytest.mark.skip("Spanner doesn't support empty inserts")
+    def test_empty_insert(self):
+        pass
```

Comment:
Also skippting a test with an empty insert. See [the docs](https://cloud.google.com/spanner/docs/dml-syntax#insert-statement).